docs/section5/Rule5-37.md

```diff
@@ -0,0 +1,64 @@
+
+# Envelope - Rule 5-37  
+
+**Rule ID:** 5-37  
+**Rule Description:** Skylight U-factors for residential, non-residential and semi-heated spaces in the baseline model must match the appropriate requirements in Table G3.4-1 through G3.4-8.  
+**Rule Assertion:** B-RMR subsurface: U_factor = expected value  
+**Appendix G Section:** Section G3.1-5(e) Building Envelope Modeling Requirements for the Baseline building  
+**Appendix G Section Reference:** Tables G3.4-1 to G3.4-8  
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:** None  
+
+**Manual Check:** Yes  
+**Evaluation Context:** Each Data Element  
+**Data Lookup:** Tables G3.4-1 to G3.4-8  
+**Function Call:**
+
+  1. get_surface_conditioning_category()  
+  2. get_opaque_surface_type()  
+  3. get_building_skylight_roof_areas()
+
+## Rule Logic:  
+
+- Get building climate zone: `climate_zone = B_RMR.weather.climate_zone`  
+
+- Get surface conditioning category dictionary for B_RMR: `scc_dictionary_b = get_surface_conditioning_category(B_RMR)`  
+
+- Get building skylight roof areas dictionary: `skylight_roof_areas_dictionary_b = get_building_skylight_roof_areas(B_RMR)`
+
+- For each building segment in the Baseline model: `for building_segment_b in B_RMR.building.building_segments:`  
+
+  - Calculate building segment skylight roof ratio: `srr_b = skylight_roof_areas_dictionary_b[building_segment_b.id][0] / skylight_roof_areas_dictionary_b[building_segment_b.id][1]`
+
```

Comment:
There is a pending 90.1 interpretation for whether a separate SRR must be used for each conditioning category (res, non-res, semiheated). Will hopefully be resolved at the next ECB call.

Reply:
The RDS is updated as per the interpretation. Skylight-to-roof ratio function is also updated accordingly.

Reply:
@xingcx and @mkarpman so should the function 'get_building_segment_skylight_roof_ratios' be 'get_building_space_conditioning_catergory_skylight_roof_ratios' ??

Reply:
I think this function should do both - provide ratios by space conditioning categories (which would be used to determine baseline skylight U/SHGC/VT) and also for all conditioning categories combined (which will be used to determine baseline skylight area). The function needs to provide thes ratios  for each unique buildingsegment.area_type_vertical_fenestration. I included comments to this effect for this function. We also

Reply:
@mkarpman it might be better to have 2 separate functions so that we're clear in the RDS what is being used where.